source/app/powertabeditor.cpp

```diff
@@ -229,7 +229,16 @@ void PowerTabEditor::openFile(QString filename)
     if (filename.isEmpty())
         return;
 
+	int validationResult = myDocumentManager->containsDocument(filename.toStdString());
```

Comment:
I'm not sure why this happens - it must be a github issue as my editor shows it as 4 spaces and in the settings it is set as 4 spaces

![capture](https://cloud.githubusercontent.com/assets/10855194/11219070/3e549914-8d50-11e5-8f2d-f5ccc0f9ad95.PNG)